### PR TITLE
Feedly(QoL changes): List Dropdown for Collections and Read Later

### PR DESCRIPTION
**Pull request by ShubhamVerma1811**
## Description

- Moved from using List.section to List.Dropdown
- Add a Save to Read later form action.

## Screencast

Screenshot 2023-03-22 at 19 13 31

Screenshot 2023-03-22 at 19 14 28

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/feedly/src/components/Feed.tsx [hunk 1]
```diff
@@ -58,12 +96,24 @@ const Feed = ({ id, title }: { id: IFeed['id']; title: IFeed['title'] }) => {
                         actions={
                           <ActionPanel>
                             <Action.OpenInBrowser url={item.canonicalUrl} />
+                            <Action.SubmitForm
```
Comment (pernielsentikaer) [inline extensions/feedly/src/components/Feed.tsx:99]: You don't have a `Form`, you should go with `<Action>` instead

File: extensions/feedly/src/components/Feed.tsx [hunk 2]
```diff
@@ -58,12 +96,24 @@ const Feed = ({ id, title }: { id: IFeed['id']; title: IFeed['title'] }) => {
...
+                              shortcut={{ modifiers: ['cmd'], key: 's' }}
+                              onSubmit={() => handleMarkAsSaved(item.id)}
+                            />
                           </ActionPanel>
                         }
                       />
                     }
                   />
                   <Action.OpenInBrowser url={item.canonicalUrl} />
+                  <Action.SubmitForm
```
Comment (pernielsentikaer) [inline extensions/feedly/src/components/Feed.tsx:111]: You don't have a `Form`, you should go with `<Action>` instead

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you 😊  

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by ShubhamVerma1811** [comment]
Hi,
I don't see the actions that is in the video.
I was not able to reproduce it.

**Comment by ShubhamVerma1811** [comment]
Hey @pernielsentikaer, Are we waiting for any changes on this?
As of the issue you mentioned, I wasn't able to repro it.

**Comment by pernielsentikaer** [comment]
Try to run it with `npm run dev` and do as above, I can still reproduce it

**Comment by ShubhamVerma1811** [comment]
Fixed it! 🕺 

@pernielsentikaer, Please review it for any other changes

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**